catroid/src/org/catrobat/catroid/ui/fragment/SpritesListFragment.java

```diff
@@ -358,6 +386,30 @@ public void startDeleteActionMode() {
 		}
 	}
 
+	private void moveSpriteDown() {
+		Collections.swap(spriteList, spritePosition + 1, spritePosition);
+		spriteAdapter.notifyDataSetChanged();
+	}
+
+	private void moveSpriteUp() {
+		Collections.swap(spriteList, spritePosition - 1, spritePosition);
+		spriteAdapter.notifyDataSetChanged();
+	}
+
+	private void moveSpriteBottom() {
+		for (int i = spritePosition; i < spriteList.size() - 1; i++) {
+			Collections.swap(spriteList, i, i + 1);
+		}
```

Comment:
This is not possible, because it is not a rotation !
For example when i have:
Item1
Item2
Item3
Item4
Item5

and use moveToTop(Item4) I get:

Item4
Item1
Item2
Item3
Item5